packages/gateway/src/gateway.js

```diff
@@ -53,6 +56,16 @@ export async function gatewayGet(request, env, ctx) {
   const cid = getCidFromSubdomainUrl(reqUrl)
   const pathname = reqUrl.pathname
 
+  if (env.DENYLIST) {
+    const anchor = await toDenyListAnchor(`${cid}/`)
+    // TODO: in theory we should check each subcomponent of the pathname also.
```

Comment:
So, this means go through the path and test all wrapping directories if they exist?

Reply:
We decided on a call that we'd not support this right now as the vast majority of blocked items are CID only.

Reply:
Maybe worth rephrasing this comment to what we decided for now, and what are the options we can consider moving to in the future? (and an issue to track it)

Reply:
I will open an issue for this as well and elaborate!

Reply:
https://github.com/nftstorage/nft.storage/issues/1737